1.8/usage/cli/index.md

```diff
@@ -87,45 +87,45 @@ $ dcos config set core.dcos_url 52.36.102.191
 ```
     
 
-**email** Your email address. This is set by default during installation. For example, to reset your email address:
+**dcos_acs_token** The token generated by authenticating to DC/OS with ACS. For example, to change the ACS token:
```

Comment:
Instead of `to change the ACS token`: `To set the ACS token, run `dcos auth login`